pkgs/applications/networking/station/default.nix

```
@@ -1,13 +1,13 @@
 { appimageTools, fetchurl, lib }:
-
 let
   pname = "station";
-  version = "1.52.2";
+  version = "v2.5.0";
```

```suggestion
  version = "2.5.0";
```
`v` is a prefix, not a version.